mirascope/core/openai/call_response.py

```diff
@@ -173,3 +188,23 @@ def tool_message_params(
             )
             for tool, output in tools_and_outputs
         ]
+
+    @computed_field
+    @property
+    def _audio(self) -> ChatCompletionAudio | None:
```

Comment:
I don't quite see the value of having this as a computed field since it's just access and not constructing or doing anything.

I think instead we should just call this directly in the `audio` and `audio_transcript` properties, and we should make both of those properties computed fields so they appear in the model dump